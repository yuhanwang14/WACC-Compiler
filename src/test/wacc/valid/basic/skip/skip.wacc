# simple skip program

# Output:

# Program:

begin skip end
