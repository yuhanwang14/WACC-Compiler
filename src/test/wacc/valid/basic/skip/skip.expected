1
2
3
4
5
6
7
8
# simple skip program

# Output:

# Program:

begin skip end
 